spec/features/applications/confirmation_is_rendered_spec.rb

```diff
@@ -64,6 +68,50 @@
       scenario 'the income label displays correctly' do
         expect(page).to have_xpath('//div[contains(@class,"summary-result") and contains(@class,"part")]', text: 'Waiting for evidence')
       end
+
+      scenario 'the grant help with fees form is not rendered' do
+        expect(page).not_to have_content 'Grant help with fees'
```

Comment:
Only the usual `.to have_no_content` instead of `not_to have_content` - small thing, which occurs here a few times.